plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -2583,3 +2583,30 @@ function wc_cache_get_multiple( $keys, $group = '', $force = false ) {
 	}
 	return $values;
 }
+
+/**
+ * Enable WebP images in WooCommerce.
+ *
+ * WebP files should be able to be imported in WooCommerce, as they've been supported in WordPress core since v5.8.
+ */
+add_filter( 'woocommerce_rest_allowed_image_mime_types', 'more_mimes_to_exts' );
```

Comment:
This filter isn't necessary if you are already adding `webp` to the array being passed to the filter.

Reply:
Just to clarify, it is suggested to remove line 2592. Correct?

Reply:
I went ahead with doing so.

Reply:
> Just to clarify, it is suggested to remove line 2592.

There is no need for the `more_mimes_to_exts` function as it was being called by the filter.